composer.json

```
@@ -1,11 +1,11 @@
 {
-  "require": {},
+  "require": {
+    "php": "^8.0",
+    "ext-pdo": "*"
+  },
   "autoload": {
     "psr-4": {
-      "controllers\\": "controllers",
-      "models\\": "models",
-      "views\\": "views",
-      "app\\": "app"
+      "Source\\": "src"

```

```suggestion
      "App\\": "src"

```
App или Mvc или Framework (в общем, какое-то название) будет лучше смотреться имхо.
но не настаиваю, просто мнение